index.d.ts

```diff
@@ -0,0 +1,12 @@
+import {Store} from 'keyv'
+
+declare const keyvFirestore: new (options: {
```

Comment:
This is actually not precise, but enough for use with keyv

Typing the entire class is not worth it IMO